data/brands/amenity/recycling.json

```diff
@@ -266,7 +265,7 @@
         "amenity": "recycling",
         "brand": "The Salvation Army",
         "brand:wikidata": "Q188307",
-        "name": "The Salvation Army",
+        "recycling_type": "container",
```

Comment:
Good catch! It should’ve been a `container` all along.

This entry is currently shared by the UK and U.S. There are a lot of other entries here for donation bins of other U.S. organizations, so there should probably be a discussion in the U.S. community about whether to keep `name` on them. In the meantime, you can copy the entry and have one be for the UK and the other for the U.S. to sidestep the broader issue.

(With as many organizations as we have running donation bins over here, people may well use the brand names as names, in much the same fashion as the gas stations they’re usually placed in front of. However, I’m pretty sure the Salvation Army donation bins go by a distinct name, something like Small Item Donation Center. I’m unsure if it varies by region.)

Reply:
Thank you for your helpful comments.  I'm happy to update my patch so that the US and GB have separate entries for Salvation Army recycling points following each country's conventions if there's general agreement that's the best approach.  What do others think?